package.xml

```diff
@@ -10,6 +10,8 @@
 
   <build_depend>catkin</build_depend>
   <build_depend>python</build_depend>
+  <build_depend version_gte="0.4.0">python-catkin-tools</build_depend>
+  <run_depend version_gte="0.4.0">python-catkin-tools</run_depend>
```

Comment:
Yeah, I guess only systematic reaction is that catkin [prints warning in a certain condition](http://answers.ros.org/question/136526/how-versioned-build_depend-works/). 
I added it more for human purpose, not for machine.